<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>50</y>
      <width>166</width>
      <height>316</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLineEdit" name="titleLineEdit">
       <property name="text">
        <string>titleLineEdit</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="descriptionLineEdit">
       <property name="text">
        <string>descriptionLineEdit</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateEdit" name="dateEdit">
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="quickAddButton">
       <property name="text">
        <string>quickAddButton</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="addButton">
       <property name="text">
        <string>addButton</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="removeButton">
       <property name="text">
        <string>removeButton</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="toggleButton">
       <property name="text">
        <string>toggleButton</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="viewCompletedButton">
       <property name="text">
        <string>viewCompletedButton</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="filterComboBox"/>
     </item>
    </layout>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>30</y>
      <width>256</width>
      <height>192</height>
     </rect>
    </property>
    <property name="columnCount">
     <number>8</number>
    </property>
    <column>
     <property name="text">
      <string>Title</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Description</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Date</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Status</string>
     </property>
    </column>
    <column/>
    <column/>
    <column/>
    <column/>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
